### PR TITLE
Authentication login

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
Good to go. One suggestion, if variable is of type constant, then use **const** instead of let.

File: css/styles.css [hunk 1]
```diff
@@ -1,10 +1,59 @@
...
+  text-decoration: none;
 }
 
 .theme-heading{
   font-family: 'Sacramento', cursive;
 }
+
+.landing-footer-section{
+  padding: 2rem;
+  /* border-top: 2px solid; */
```
Comment (sanjitsarkar) [inline css/styles.css:21]: remove unused code

**Review by sanjitsarkar — APPROVED** [review]

File: css/styles.css [hunk 2]
```diff
@@ -1,10 +1,59 @@
 @import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,700;1,800;1,900&family=Sacramento&display=swap');
 @import 'color.css';
 
-html{
+:root{
   font-family: 'Barlow Condensed', sans-serif;
+  background-color: var(--background);
+  font-weight: 200;
+
```
Comment (b0llu) [inline css/styles.css:8]: you can remove extra space

File: js/homepage.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (b0llu) [inline js/homepage.js:1]: Good written JS file easily understandable

File: templates/login.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            <i class="fa-brands fa-twitter f-l"></i>
+
+          </div>
+
+        </div>
+
+      </article>
+    </section>
+  </section>
+
```
Comment (manojsarna) [inline templates/login.html:67]: check for consistent spacing

File: templates/home.html [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+</head>
+<body>
+  <header>
+    <nav class="nav">
+      <div class="start">
+        <h2 class="theme-heading primary-col">Thrify with me</h2>
+        <i class="fas fa-bars primary-col f-m" id="side-menu-btn"></i>
+      </div>
+      <ul class="list list-inline f-m">
+        <li class=''>
```
Comment (b0llu) [inline templates/home.html:19]: the class field is empty here

File: templates/login.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  <section class="grid-three-layout-container">
+    <section class="grid-three-layout-center">
+      <article class="card card-shadow l">
+        <div class="background">
+          <img src="/media/auth/login.jpg" alt="">
+        </div>
+        <div class="card-content stacked">
+          <h2 class="center primary-col f-xl fw-li">Login</h2>
+          <!-- <div class="card-content stacked"> -->
```
Comment (manojsarna) [inline templates/login.html:42]: can use webp instead of jpg or png for better performance , webpage will load faster

File: templates/login.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+            <input placeholder=" " class="input w-m" type="email">
+            <label class="input-label f-s fw-li">Email</label>
+          </div>
+          <div class="input-container ">
+            <input placeholder=" " class="input w-m" type="password">
+            <label class="input-label f-s fw-li">Password</label>
+          </div>
+
+          <a href="#" class="btn f-s primary center">Login</a>
+
```
Comment (manojsarna) [inline templates/login.html:53]: Use placeholder for guiding your end user , not empty string!!

File: templates/home.html [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+            <div class="input-container ">
+              <input placeholder=" " class="input w-full">
```
Comment (b0llu) [inline templates/home.html:21]: if you are using a placeholder dont leave it empty

File: templates/home.html [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+            <li>Youtube</li>
+          </ul>
+        </div>
+    </article>
+
+    <article class="grid-three-layout-right landing-footer-section">
+      <div class="">
+        <p class="f-s">Reach out to us</p>
+        <p>Industrial Textile</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, </p>
```
Comment (manojsarna) [inline templates/home.html:175]: class="f-s" ? come up with good names !!

File: templates/home.html [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+          <p>sed do eiusmod tempor incididunt ut labore sunt in culpa </p>
+          <p>qui officia deserunt mollit anim id est laborum.</p>
+        </div>
+
+      </article>
+
+      <article class="grid-three-layout-center landing-footer-section">
+        <div class="">
+          <p class="f-l">Join our news letter</p>
+          <ul class="list list-inline">
```
Comment (manojsarna) [inline templates/home.html:163]: cant understand what is f-l , f-s

File: templates/home.html [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+        </article>
+        </div>
+    </section>
+</section>
+
+<hr class="primary-col">
+    <footer class="grid-three-layout-container">
+
+      <article class="grid-three-layout-left landing-footer-section">
+        <div class="">
```
Comment (b0llu) [inline templates/home.html:150]: class field is empty here

File: templates/home.html [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+          <p>Industrial Textile</p>
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, </p>
+          <p>sed do eiusmod tempor incididunt ut labore sunt in culpa </p>
+          <p>qui officia deserunt mollit anim id est laborum.</p>
+        </div>
+
+      </article>
+
+      <article class="grid-three-layout-center landing-footer-section">
+        <div class="">
```
Comment (b0llu) [inline templates/home.html:161]: class field is empty here

File: templates/home.html [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+
+<section class="slider-bottom">
+  <h2 class="center-txt fw-reg primary-col">Footwear</h2>
+    <section class="grid-two-layout-container">
+      <div class="container fl-end">
+        <article class="grid-two-layout-left">
+            <div class="card w-xl">
+              <img src="/media/home/left-bottom-1.webp"  id="slid-b-img-left"  alt="" class="background">
+              <div class="card-content stacked">
+                <h3 class="fw-li">Women</h3>
```
Comment (manojsarna) [inline templates/home.html:124]: Good , do that everywhere!!

File: templates/home.html [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+          <ul class="list list-inline">
+            <li>Instagram</li>
+            <li>Facebook</li>
+            <li>Youtube</li>
+          </ul>
+        </div>
+    </article>
+
+    <article class="grid-three-layout-right landing-footer-section">
+      <div class="">
```
Comment (b0llu) [inline templates/home.html:172]: class field is empty here

File: templates/login.html [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+</head>
+
+<body id="login-bg">
+  <header>
+    <nav class="nav">
+      <div class="start">
+        <h2 class="theme-heading primary-col">Thrify with me</h2>
+      </div>
+      <ul class="list list-inline f-m">
+        <li class=''>
```
Comment (b0llu) [inline templates/login.html:20]: class field is empty here

File: templates/login.html [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+          <div class="input-container ">
+            <input placeholder=" " class="input w-full">
```
Comment (b0llu) [inline templates/login.html:22]: placeholder is empty

File: templates/login.html (hunk 6 above)
Comment (b0llu) [inline templates/login.html:42]: remove commented code

File: templates/home.html [hunk 17]
```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Thrift</title>
+  <link rel="stylesheet" href="https://traquil-ui.netlify.app/styles.css">
+  <link rel="stylesheet" href="/css/styles.css">
+    <script src="https://kit.fontawesome.com/854a67f594.js" crossorigin="anonymous"></script>
+</head>
```
Comment (manojsarna) [inline templates/home.html:10]: script tags at the end of body !!

File: templates/login.html [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+    <section class="grid-three-layout-center">
+      <article class="card card-shadow l">
+        <div class="background">
+          <img src="/media/auth/login.jpg" alt="">
+        </div>
+        <div class="card-content stacked">
+          <h2 class="center primary-col f-xl fw-li">Login</h2>
+          <!-- <div class="card-content stacked"> -->
+          <div class="input-container ">
+            <input placeholder=" " class="input w-m" type="email">
```
Comment (b0llu) [inline templates/login.html:44]: placeholder empty

File: js/homepage.js [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+  let imgPath="/media/home/";
+  let slider1Fwd= document.getElementById('slid-t-btn-fwd');
+  let slider2Fwd= document.getElementById('slid-m-btn-fwd');
+  let slider3Fwd= document.getElementById('slid-b-btn-fwd');
+  let slider1Rev= document.getElementById('slid-t-btn-rev');
+  let slider2Rev= document.getElementById('slid-m-btn-rev');
+  let slider3Rev= document.getElementById('slid-b-btn-rev');
+  let sideMenuBtn= document.getElementById('side-menu-btn');
+  let sideMenu= document.getElementById('side-menu');
+let currentPg= 1;
```
Comment (manojsarna) [inline js/homepage.js:11]: use const instead of let

File: templates/login.html [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+            <label class="input-label f-s fw-li">Email</label>
+          </div>
+          <div class="input-container ">
+            <input placeholder=" " class="input w-m" type="password">
```
Comment (b0llu) [inline templates/login.html:48]: check here also

File: templates/login.html [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+            <label class="input-label f-s fw-li">Password</label>
+          </div>
+
+          <a href="#" class="btn f-s primary center">Login</a>
```
Comment (b0llu) [inline templates/login.html:52]: no link connected href

**Review by b0llu — APPROVED** [review]
suggested few changes do try to implement them. Everything else looks good

**Review by manojsarna — COMMENTED** [review]
Overall the code looks good!!

Suggested Some Changes , Do Check!!

File: templates/login.html (hunk 6 above)
Comment (ritikaawasthy) [reply templates/login.html:42]: changed to webp

File: templates/home.html (hunk 10 above)
Comment (ritikaawasthy) [reply templates/home.html:163]: font large , font small

**Pull request merged**